src/main/java/br/com/harbitech/school/alternative/Alternative.java

```diff
@@ -1,57 +1,33 @@
 package br.com.harbitech.school.alternative;
 
 import br.com.harbitech.school.question.Question;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
 
 import javax.persistence.*;
 
-import static br.com.harbitech.school.validation.ValidationUtil.validateNonBlankText;
-import static br.com.harbitech.school.validation.ValidationUtil.validateNonNullClass;
 @Entity
 public class Alternative {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(columnDefinition = "TEXT")
    private String text;
```

Comment:
É bom um `@NotBlank` aqui!

```suggestion
   @NotBlank
   private String text;
```